src/Services/Flight/src/Flight/Data/Migrations/20230526093041_aircraftValueObject.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
```

Comment:
Why we have empty migration? It should be deleted.

Reply:
I'll work on implementing the recommended changes.